.github/workflows/release.yml

```diff
@@ -39,29 +39,21 @@ jobs:
           bazel test --local_ram_resources=4096 -c opt -k --test_timeout 300,450,1200,3600 --test_output=errors //tensorflow_recommenders_addons/...
   release-wheel:
     name: Build release wheels
-    runs-on: ${{ matrix.os }}
     strategy:
       matrix:
         # TODO: add back 'windows-latest' when it can be compiled.
         os: ['macos-12', 'ubuntu-20.04']
         py-version: ['3.7', '3.8', '3.9']
-        tf-version: ['2.6.3', '2.8.3']
-        tf-need-cuda: ['1', '0']
+        tf-version: ['2.6.3', '2.11.0']
```

Comment:
Considering the min version TF here is only for UT, we may change it to '2.8.3'.

Reply:
Changed the min version to 2.8.3